modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -1887,66 +1887,22 @@ public void testStagingAware() throws Exception {
 	@Test
 	public void testTextMatchOverMultipleFields_bestFields() throws Exception {
 		_journalArticleBuilder.setTitle(
-			"drink carbonated coca"
-		).setContent(
-			"carbonated cola"
-		).build();
-
-		_journalArticleBuilder.setTitle(
-			"drink carbonated pepsi cola"
-		).setContent(
-			"carbonated cola cola"
-		).build();
-
-		_journalArticleBuilder.setTitle(
-			"fruit punch"
-		).setContent(
-			"non-carbonated cola"
+			"coca cola"
 		).build();
 
 		_journalArticleBuilder.setTitle(
-			"sprite"
+			"coca coca"
 		).setContent(
-			"carbonated cola cola"
+			"cola cola"
 		).build();
 
-		_getTextMatchOverMultipleFields();
-
 		_updateElementInstancesJSON(
 			new Object[] {_getTextMatchOverMultipleFields()},
 			new String[] {"Text Match Over Multiple Fields"});
 
 		_keywords = "coca cola";
 
-		_assertSearch(
-			"[drink carbonated coca, drink carbonated pepsi cola, sprite, " +
-				"fruit punch]");
-
-		_journalArticleBuilder.setTitle(
-			"lorem ipsum dolor"
-		).setContent(
-			"ipsum sit"
-		).build();
-
-		_journalArticleBuilder.setTitle(
-			"lorem ipsum sit"
-		).setContent(
-			"ipsum sit sit"
-		).build();
-
-		_journalArticleBuilder.setTitle(
-			"nunquis"
-		).setContent(
-			"non-lorem ipsum sit"
-		).build();
-
-		_updateElementInstancesJSON(
-			new Object[] {_getTextMatchOverMultipleFields()},
-			new String[] {"Text Match Over Multiple Fields"});
-
-		_keywords = "ipsum sit sit";
-
-		_assertSearch("[lorem ipsum sit, lorem ipsum dolor, nunquis]");
+		_assertSearch("[coca cola, coca coca]");
```

Comment:
Could you add an assertSearch without any elements to make sure this test is actually working?

@felipelorenz @AlmirFe